src/utils/apiClient.tsx

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios';
+
+const baseUrl = `${process.env.REACT_APP_API_URL ?? ''}api/`;
+
+const apiClient = axios.create({
+  baseURL: baseUrl,
```

Comment:
@sniedzielski If `process.env.REACT_APP_API_URL` will be set in the `.env` won't it colide with the proxy set in the `package.json`? Perhaps it would be better to remove the proxy and move `http://localhost:8000` into a `.env` and set as a default `REACT_APP_API_URL` which would be simply swapped whenever it's needed?

Reply:
I will remove 'proxy' and make 'localhost:8000' as a default value for 'REACT_APP_API_URL' variable. It sounds better than current solution.